functions

```diff
@@ -390,6 +390,7 @@ _add_module() {
                 ;;
             filename)
                 quiet "filename: %s" "$value"
+                [[ "$value" == '(builtin)' ]] && continue
```

Comment:
Because this is a string madeup by kmod, I'm disinclined to depend on it. Can't we check that it looks like a path (similar to what I did)?

Reply:
Because (builtin) is a madeup string kmod sticks to builtin modules now, we can rely on builtin modules having it and skipping them. If kmod reports a non-path like string, we should not work around it.

Reply:
My point is that it might change in the future. Or, maybe kmod adds some other new madeup string to ascribe to some other kind of pseudo-module. In that case, we'd have to change our current exception or add a new one. Why bother trying to add things that don't look like paths?

Reply:
If kmod breaks this in the future, we can start checking for path-like strings. For now this is the only exception. If the filename does not look like a path outside of this, the bug is in the wrongly reported filename.